tests/AttributeBagTest.php

```
@@ -103,39 +83,40 @@ public function testRemove(array $attributes, string|UnitEnum $key, array $expec
+     * @phpstan-param mixed[] $attributes
+     * @phpstan-param mixed[] $values
+     * @phpstan-param mixed[] $expected
+     */
+    #[DataProviderExternal(AttributeBagProvider::class, 'setMany')]
+    public function testSetMany(array $attributes, array $values, array $expected): void
     {
-        $attributes = ['id' => 'submit'];
-
-        $this->expectException(InvalidArgumentException::class);
-        $this->expectExceptionMessage(Message::KEY_MUST_BE_NON_EMPTY_STRING->getMessage());
+        AttributeBag::setMany($attributes, $values);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\AttributeBag' in method 'testSetMany'. (undefined)

(StaticAccess)